lib/check-version.js

```diff
@@ -36,3 +36,17 @@ module.exports = function (done) {
     done()
   })
 }
+
+function warnForVue2Version (template, name) {
+  var vue1InitCommand = 'vue init ' + template + '#1.0' + ' ' + name
+
+  console.log(chalk.red('  This will install Vue 2.x version of template.'))
+  console.log()
+  console.log(chalk.yellow('  For Vue 1.x use: ') + chalk.green(vue1InitCommand))
+  console.log()
+}
+
+module.exports = {
+  checkVersion: checkVersion,
+  warnForVue2Version: warnForVue2Version
+}
```

Comment:
Wondering if it would be better to just create a new file for `warnForVue2Version`. It doesn't have anything to do with checking the node version, so having it in here might make it difficult to find in a quick search. What do you think?